mkdocs.yml

```diff
@@ -45,7 +45,9 @@ nav:
     monoDrive_home/Ultrasonic.md,
     monoDrive_home/Viewport.md]
   - Simulator : monoDrive_home/Simulator-Configuration.md
-  - Vehicle : monoDrive_home/Vehicle-Configuration.md
+  - Vehicle : [
+    monoDrive_home/Vehicle-Configuration.md,
+    monoDrive_home/CarSim.md]
```

Comment:
I think that instead of adding this as a main section within the Simulator, we should just mention it as a note under Vehicles / Vehicle Dynamics with a link. E.g.
"Note that monoDrive also support co-simulation with CarSim for vehicle dynamics. For more information, please see this tutorial."

Then all of this can live as a tutorial under some new "Co-simulation Tutorials" section. You could embed or link that video there as well.